packages/flutter/test/material/app_bar_test.dart

```diff
@@ -3167,6 +3167,37 @@ void main() {
     expect(titleOffset.dx, backButtonOffset.dx + titleSpacing);
   });
 
+  testWidgets('AppBar actions padding can be adjusted', (WidgetTester tester) async {
+    final Key appBarKey = UniqueKey();
+    final Key actionKey = UniqueKey();
+
+    Widget buildAppBar({ EdgeInsetsGeometry? actionsPadding }) {
+      return MaterialApp(
+        home: Scaffold(
+          appBar: AppBar(
+            key: appBarKey,
+            actions: <Widget>[
+              SizedBox.square(key: actionKey, dimension: 40.0),
+            ],
+            actionsPadding: actionsPadding,
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildAppBar());
+
+    // Test actions position without actions padding.
```

Comment:
```suggestion
    // Actions padding default to zero padding.
```
Just a suggestion, if you think it is better.